lib/kitchen/driver/openstack.rb

```diff
@@ -372,29 +368,6 @@ def hints_path
         Ohai::Config[:hints_path][0]
       end
 
-      def setup_ssh(server, state)
-        if config[:key_name]
-          info "Using OpenStack keypair <#{config[:key_name]}>"
-        end
-        info "Using public SSH key <#{config[:public_key_path]}>"
-        info "Using private SSH key <#{config[:private_key_path]}>"
-        state[:ssh_key] = config[:private_key_path]
-        do_ssh_setup(state, config, server) unless config[:key_name]
-      end
-
-      def do_ssh_setup(state, config, server)
```

Comment:
Really? Ah, what i bet is happening is `cloud-init` is injecting the key for me or something. So maybe I should keep this around. I'll have to investigate this farther.